imports/plugins/core/layout/client/templates/layout/admin/admin.js

```diff
@@ -68,6 +68,13 @@ Template.coreAdminLayout.helpers({
     return items;
   },
 
+  isAdmin() {
+    if (Object.keys(Meteor.user().roles).length < 2) {
```

Comment:
`return Object.keys(Meteor.user().roles).length < 2` should work just fine here too